# **Curriculum vitae**

## Contact

e-mail: giannos.louloudis@gmail.com \
address: Mariendalsvej 57D, 2000. Frederiksberg, Denmark

## **Personal profile**

I am 25 years old and I come from Crete, Greece.
I have a BSc degree in Biology at the Aristotle University of Thessaloniki.
I am currently a second-year international MSc student in the Masterâ€™s in Bioinformatics at the department of Science of Copenhagen University (KU).

During my Bachelor's studies, I focused on bioinformatics, biostatistics, data mining and subsequent analyses used in the field. During the first year of my MSc I chose courses to increase my knowledge in statistics and machine learning as well as their application in biology and more specifically genomics.

## **Work experience**

### **Academic Research Assistant**

I am currently employed as an academic research assistant in the Brunak group in the Center for Protein Research (CPR). The group focuses on *Translational Disease Systems in Biology*, and I am a member of the *genetics infrastructure sub-group* which focuses on the development of software which is used by the rest of the group. My major projects up to this point have been:

* Development of an in-house PheWAS-pipeline, along with a local-browser application which visualizes the results post-analysis to aid data exploration. The software utilizes patient gentic data along with patient health records classified in the form of both ICD-10 and PheCodes.
* Use of large genetic cohorts such as the Danish Blood Donor Study and Copenhagen Hospital Biobank to find associations between patients' genetic background and covid-19 severity after infection.

### **Teaching Assistant**

After my first year of MSc studies, I applied for and became a teaching assistant for the courses that I had previously taken and was enthusiastic about. This choice was derived from my interest in sharing my enthusiasm with others on topics that fascinated me, but also an indirect goal was developing my abilities to explain complex concepts as efficiently as possible. the courses that I chose were:

| Course title                                | Academic period   | Concepts taught                     |
|:--------------------------------------------|:------------------|:------------------------------------|
| Introduction to Python for Data Science     | Block 1           | Python, BASH                        |
| Introduction to Python for Computer Science | Block 2           | Python, BASH                        |
| Introduction to Data Science                | Block 3           | Python, Statistics, ML algorithms   |

## **Education**

### **University studies**

These are the courses that I selected from my **Master's programme**, along with their corresponding grade. The grades are written in the Danish 7-point grading system:

* Computer Science:
  * Python programming for data science: 12
  * Introduction to data science: 12
  * Statistics for bioinformatics: 10
  * Machine Learning (DIKU): 7
* Computational Biology:
  * Biological sequence analysis: 12
  * Structural bioinformatics: 10
  * Bioinformatics project: 12
  * Bioinformatics of high-throughput analysis: 12

Regrading my **Bachelor's programme**:

* I have completed a four-year bachelor's degree in Biology at the Aristotle University of Thessaloniki:
  * My grade average was 8.11/10
  * I specialized in Molecular Biology, Genetics and Biotechnology.
* During my degree I did a year-long thesis along with as two-month and a six-month internship (in Turku, Finland).
* During 2018-2019 I did my **thesis/dissertation** at the lab of population genetics under the supervision of Alexander Triandafyllidis. The thesis was split into two different and unrelated projects.
  * The first project involved the development of a custom pipeline for the identification of Single Nucleotide Polymorphisms from New Generation Sequencing, Whole Genome sequencing data of *Canis lupus familiaris* (specifically the breed Cretan tracer). The piece of software I developed, takes as input raw paired end data files and quality checks them, trims if needed, maps them on a provided reference genome, does variant calling (keeping only SNPs) and finally, ends with SNP-filtering.
    * The SNPs found at the end of the pipeline were compared to those of an online database, and then using a Variant Effect Predictor tool (VEP), I checked how they affected the gene expression.
  * The other project used a pre-made pipeline, named dDocent, to extract SNPs from population data (both Paired-End and Single-End) from the organisms *Perdix perdix* and *Sparus aurata*.

### **Internship at HCMR**

* I worked for 2 months at the Hellenic Center of Marine Research in Crete, Greece and focused on developing a pipeline used for mining long non-coding RNAs (lncRNAs) from a pool of transcriptomics data.
* The lncRNAs found were examined later to check whether they contributed or not on gender determination for each species.

### **Erasmus+ Internship**

* I completed a six-month long internship at the Turku University in Finland, where along with my supervisor Spiros Papakostas worked on a genomics-transcriptomics project from start to finish.
* The project focused on locating islands of genomic inversions using custom pipelines that I wrote myself. The goal was to connect those aforementioned islands with genetic divergence between three different species of a rotifer organisms.
* With the help of my supervisor we are planning on publishing the results on a scientific journal.
